package.json

```diff
@@ -1,9 +1,9 @@
 {
   "name": "@groveco/backbone.store",
   "main": "dist/index.js",
-  "version": "0.3.2",
+  "version": "0.3.3",
```

Comment:
Do you mind leaving the version for a separate PR? So we have a explicit "intent to release" branch...
```suggestion
  "version": "0.3.2",
```

Reply:
Tyvm!